src/modules/EventModule.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+#include <qtimer.h>
+#include "core/Module.h"
+#include "UmikoBot.h"
+
+class EventModule : public Module
+{
+public:
+	struct RaffleDraw
+	{
+	public:
+		snowflake_t m_UserId;
+		QList<unsigned int> m_TicketIds;
+		RaffleDraw(snowflake_t userID)
+			:m_UserId(userID){}
+	};
+
+	struct EventConfig
+	{
+		QTimer* eventTimer{ nullptr };
+		QTimer* raffleDrawRewardClaimTimer{ nullptr };
+		bool isEventRunning{ false };
+		bool eventHighRiskHighRewardRunning{ false };
+		bool eventLowRiskLowRewardRunning{ false };
+		bool claimedReward = true;
+		bool eventRaffleDrawRunning{ false };
+		unsigned int currentTicketIndex{ 0 };
```

Comment:
Edit: Okay I see how you use it, but it could be named better. You could go with something like `numticketsDispensed` or `numticketsBought` to signify that it's the total number of tickets that have been bought.